docs/querying/caching.md

```diff
@@ -82,3 +82,19 @@ Note that the task executor processes only support caches that keep their data l
 This restriction exists because the cache stores results at the level of intermediate partial segments generated by the
 ingestion tasks. These intermediate partial segments will not necessarily be identical across task replicas, so
 remote cache types such as `memcached` will be ignored by task executor processes.
+
+## Unsupported queries
+
+Query caching is not available for following:
+- Queries, that involve a `union` datasource, do not support result-level caching. Refer to the 
+[related github issue](https://github.com/apache/druid/issues/8713) for details. Top level union SQL queries can still 
```

Comment:
```
    ../docs/querying/caching.md
       90 | [related github issue](https://github.com/apa 
>> 1 spelling error found in 167 files
```

The CI is failing because of this line. Please add a suppression in `website/.spelling`. BTW, I think it should be `GitHub`.